cmd/main.go

```diff
@@ -1,53 +1,60 @@
 package main
 
 import (
-	"TicTacGo/internal/board"
-	"fmt"
+	"fmt" // first stdlib imports
+
+	// after empty line 3rd-party imports
+
+	// always use full import paths
+	// imternal package is needed only for packages that is explicitly forbidden to use from outside
+	"github.com/Kwbmm/TicTacGo/board" // after empty line local module imports
 )
 
 func main() {
 	fmt.Println("Welcome to TicTacGo")
 
 	currentPlayer := 'O'
 	nextPlayer := 'X'
-	board := board.Board{BoardState: [9]board.Cell{
-		board.Cell{Value: '1'},
-		board.Cell{Value: '2'},
-		board.Cell{Value: '3'},
-		board.Cell{Value: '4'},
-		board.Cell{Value: '5'},
-		board.Cell{Value: '6'},
-		board.Cell{Value: '7'},
-		board.Cell{Value: '8'},
-		board.Cell{Value: '9'},
-	}}
+
+	// it's better not to use variable name the same as package name
+	// too complex struct for simple task
+	var b board.Board
```

Comment:
Would it be possible to alias `board.Board` to just `Board` like you do in Python or C++?

Python: `import board.Board as Board`
C++:
```cpp
using board;
Board b = Board()
```

Reply:
No, in go alway must reach package members through package name.